libcxx/include/__memory/unique_ptr.h

```diff
@@ -672,8 +675,8 @@ operator<=>(const unique_ptr<_T1, _D1>& __x, const unique_ptr<_T2, _D2>& __y) {
 #endif
 
 template <class _T1, class _D1>
-inline _LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX23 bool
-operator==(const unique_ptr<_T1, _D1>& __x, nullptr_t) _NOEXCEPT {
+inline _LIBCPP_HIDE_FROM_ABI
+_LIBCPP_CONSTEXPR_SINCE_CXX23 bool operator==(const unique_ptr<_T1, _D1>& __x, nullptr_t) _NOEXCEPT {
```

Comment:
These 2 lines are just reformatted. Are compilers already warning on discarding comparison results by default?

Reply:
I don't know if the warnings are enabled by default but I got warnings about unused resaults in comparisons operators when testing and adding [[nodiscard]] didn't change the warning message, also I think the library doesn't apply [[nodiscard]] to comparisons operators so I reverted the changes.